docker/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+export GITHUB_PRIVATE_KEY=/tmp/ssh_key
+
+function generate_keys {
+    ssh-keygen -b 4096 -N '' -f $GITHUB_PRIVATE_KEY
+
+    echo "New keys has been generated, below is the public key"
+    cat "$GITHUB_PRIVATE_KEY.pub"
+
+    sleep 5s
```

Comment:
It only generates the key if it doesn't exist. As long as you don't remove/destroy the container, it should be there.